### PR TITLE
Creating with ajax assignment

### PR DESCRIPTION
**Pull request by rayyhum777**

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -2,6 +2,8 @@ class Comment < ActiveRecord::Base
   belongs_to :post
   belongs_to :user
   
+  default_scope { order('updated_at DESC').reverse_order }
```
Comment (barnes7td) [inline app/models/comment.rb]: why isn't this just

```ruby
default_scope { order('updated_at ASC') }
```

**Comment by barnes7td** [comment]
Reviewed. Left one comment. The rest looks good.

File: app/models/comment.rb (hunk 1 above)
Comment (rayyhum777) [inline app/models/comment.rb]: Ah. So DESC means DESCEND. I never learned that anywhere. Thanks!

**Comment by rayyhum777** [comment]
Now I understand. Thanks, TIm!

**Comment by rayyhum777** [comment]
Hi Tim, I know you're busy, but please check my latest checkpoint and
assignment completion. Thank you!
BTW, what should I do with the old pull requests? No mentor ever got to
check them.

Have a good week

On Tue, Feb 10, 2015 at 4:51 PM, Timothy Barnes <notifications@github.com>
wrote:

> Reviewed. Left one comment. The rest looks good.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rayyhum777/bloccit/pull/4#issuecomment-73816437>.
>



-- 
Best Regards,
Ray

**Pull request merged**